Bio/Application/__init__.py

```diff
@@ -539,7 +539,6 @@ class _Option(_AbstractParameter):
     take a value, use the _Switch object instead.
 
     Attributes:
-
```

Comment:
As an aside, all these old-style docstrings using lower-case o as a bullet point need updating at some point, e.g. RST bullet points would be simplest.